test/e2e/multihoming.go

```diff
@@ -33,9 +33,9 @@ import (
 )
 
 const (
-	PolicyForAnnotation     = "k8s.v1.cni.cncf.io/policy-for"
-	nodeHostnameKey         = "kubernetes.io/hostname"
-	fromHostSubnet          = "host" // tells the test to generate IPs from the host subnet.
+	PolicyForAnnotation = "k8s.v1.cni.cncf.io/policy-for"
+	nodeHostnameKey     = "kubernetes.io/hostname"
+	fromHostSubnet      = "host" // tells the test to generate IPs from the host subnet.
```

Comment:
nit: this change should be in the `E2E localnet: make IP request for localnet pod extensible` but its also showing up in `E2E localnet: Fix requirement on number of schedulable nodes`